pkg/apiserver/apiserver.go

```diff
@@ -289,12 +289,19 @@ func (s *PorchServer) Run(ctx context.Context) error {
 
 	certStorageDir, found := os.LookupEnv("CERT_STORAGE_DIR")
 	if found && strings.TrimSpace(certStorageDir) != "" {
-		if err := setupWebhooks(ctx, webhookNs, certStorageDir); err != nil {
+		// check if we should use cert manager webhook setup
+		useCertManWebhook := false
+		// if we dont find the env var or the var is not true
+		CertManWebhook, found := os.LookupEnv("CERT_MAN_WEBHOOKS")
```

Comment:
Not sure about the EnvVar naming. CertMan is somewhat separate to the actual webHook setup.
There may be 2 use cases here. Use/dont use cert man. Define external Webhook vs internal creation.

Reply:
- could you suggest a more suitable (EnvVar) name along with a (go variable) name for these?
- i understand the need for separation as the code is not really interacting with cert manager. 
- would ExternalWebhook & useExternalWebhook do fine?

Reply:
I agree with @efiacor.
In my understanding the webhook itself is the same: it validates if a PackageRevision can be deleted or not. The main change you introduced is that the certificate used to serve the webhook is coming from cert-manager, instead of a locally created self-signed cert. Is my understanding correct?
If it is, then I propose to use  "USE_CERT_MAN_FOR_WEBHOOK" as the name of the environment variable.

Reply:
- regarding the first note yes your understanding is correct. all i am introducing is the ability to use externally defined web-hooks along with the relevant authentication for these web-hooks such as using cert manager to both create certs and update them when they are about to expire automatically.
- Regarding the variable name i agree also, i was just looking for advice on an appropriate name :), i will use "USE_CERT_MAN_FOR_WEBHOOK"